app/services/katello/ui_notifications/subscriptions/expire_soon.rb

```diff
@@ -0,0 +1,56 @@
+module Katello
+  module UINotifications
+    module Subscriptions
+      class ExpireSoon
+        class << self
+          def deliver!
+            subs_expiring_soon.each do |subscription|
+              next if notification_already_exists?(subscription)
+              ::Notification.create!(
+                :subject => subscription,
+                :initiator => User.anonymous_admin,
+                :audience => Notification::AUDIENCE_ADMIN,
```

Comment:
I agree, but how do you set this? `AUDIENCE_TAXONOMY` is `subject.user_ids` - if `subject` is the subscription, it will not work. The `subject` can't be the taxonomy